### PR TITLE
Fixes/changes necessary while running JPMC tests.

### PR DESCRIPTION
**Pull request by mpereira**

### CODE REVIEW

**Comment by mpereira** [comment]
> This does not include the changes required to the Scala files though?

Correct. We discovered that the Scala files would need changing *with* these changes. I'll get them done as soon as I get to the office, unless you feel like working on it.

File: testing/sdk_utils.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ************************************************************************
 '''
 import functools
+import operator
```
[the remark this answers is not in the document]
Comment (mpereira) [reply testing/sdk_utils.py]: `dcos-commons` already has this, I just missed it when copy-pasting.

File: scale-tests/README.md [hunk 2]
```diff
@@ -1,15 +1,62 @@
...
+```bash
+AWS_ACCESS_KEY='<aws_access_key>'
+AWS_SECRET_KEY='<aws_secret_key>'
+AWS_ACCESS_TOKEN='<aws_access_token>'
+S3_BUCKET=infinity-artifacts/scale-tests
+SPARK_BUILD_PATH="${HOME}/mesosphere/spark-build"
+JAR_LOCAL_PATH="${SPARK_BUILD_PATH}/tests/jobs/scala/target/scala-2.11/dcos-spark-scala-tests-assembly-0.2-SNAPSHOT.jar"
+JAR_NAME="dcos-spark-scala-tests-assembly-$(date +%Y%m%d)-$(git rev-parse --short HEAD).jar"
+
+s3cmd put \
```
[the remark this answers is not in the document]
Comment (elezar) [reply scale-tests/README.md]: *UPDATE:* command changed.

**Comment by elezar** [comment]
No tests or changes to Spark code in this PR.

Only README and scale-tests.

**Pull request merged**